cloudflare/resource_cloudflare_spectrum_application_test.go

```diff
@@ -0,0 +1,287 @@
+package cloudflare
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"os"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccCloudflareSpectrumApplication_Basic(t *testing.T) {
+	var spectrumApp cloudflare.SpectrumApplication
+	zone := os.Getenv("CLOUDFLARE_DOMAIN")
+	rnd := acctest.RandString(10)
+	name := "cloudflare_spectrum_application." + rnd
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudflareSpectrumApplicationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCloudflareSpectrumApplicationConfigBasic(zone, rnd),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCloudflareSpectrumApplicationExists(name, &spectrumApp),
+					testAccCheckCloudflareSpectrumApplicationIDIsValid(name),
+					resource.TestCheckResourceAttr(name, "protocol", "tcp/22"),
+					resource.TestCheckResourceAttr(name, "origin_direct.#", "1"),
+					resource.TestCheckResourceAttr(name, "origin_direct.0", "tcp://120.120.102.10:23"),
```

Comment:
I think we should use reserved IP address, instead of random public one.
See: https://en.wikipedia.org/wiki/Reserved_IP_addresses
```suggestion
					resource.TestCheckResourceAttr(name, "origin_direct.0", "tcp://192.0.2.1:23"),
```